### PR TITLE
[plugin.video.nrk] v4.8.0

### PR DESCRIPTION
**Pull request by andrea99boban**
### Checklist:

- [ x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x ] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [ x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Comment by basrieter** [comment]
See discussion with original maintainer here: https://github.com/xbmc/repo-plugins/pull/4289

File: plugin.video.nrk/addon.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <extension point="xbmc.addon.metadata">
+    <platform>all</platform>
+    <language>no</language>
+    <summary lang="en">Watch NRK Web TV</summary>
+    <summary lang="no">Se NRK Nett-TV</summary>
+    <summary lang="it">Guarda NRK (in Norvegia)</summary>
+    <description lang="en">NRK Web TV gives you the opportunity to watch live TV as well as recordings of many shows.</description>
+    <description lang="no">NRK Nett-TV gir deg muligheten til å se mange av NRKs programmer både direkte og i opptak.</description>
+    <description lang="it">Questo addon per NRK rende possibile la visione di molti programmi di NRK sia in diretta che in differita.</description>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 3, 29 June 2007</license>
```
Comment (basrieter) [inline plugin.video.nrk/addon.xml]: Could you please use the corresponding SPDX license identifier: https://spdx.org/licenses/

**Review by basrieter — CHANGES_REQUESTED** [review]
Just a minor thing in the addon.xml that we are trying to straighten out all over the add-ons.

File: plugin.video.nrk/addon.xml (hunk 1 above)
Comment (andrea99boban) [reply plugin.video.nrk/addon.xml]: no problem, done

**Review by basrieter — APPROVED** [review]

**Pull request merged**